### PR TITLE
chapter 7.1 translated, pending review

### PR DESCRIPTION
**Pull request by icculp**

### CODE REVIEW

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+
+Si elige usar el atajo de `walletcreatefundedpsbt`, así es como se ve en su lugar:
+
+![](images/psbt-roles-for-cli-2.png)
+
+Finalmente, si en cambio necesita más control y elige usar `utxoupdatepsbt` (que no está documentado en gran parte aquí), en su lugar tiene este flujo de trabajo:
+
+![](images/psbt-roles-for-cli-3.png)
+
+## Resumen: creación de una transacción de Bitcoin firmada parcialmente
```
Comment (csralvall) [inline es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:503]: `## Resumen: creando una transacción de Bitcoin firmada parcialmente`

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 2]
```diff
@@ -0,0 +1,509 @@
...
+
+La creación de un PSBT implica un flujo de trabajo algo complejo de creación, actualización, firma, finalización y extracción de un PSBT, después de lo cual se convierte nuevamente en una transacción sin procesar. ¿Por qué se tomaría tanto trabajo? Porque quiere colaborar entre múltiples usuarios o múltiples programas. Ahora que comprende este flujo de trabajo, la siguiente sección tiene algunos ejemplos reales de cómo hacerlo.
+
+## Que sigue?
```
Comment (csralvall) [inline es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:507]: `## ¿Que sigue?`

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 3]
```diff
@@ -0,0 +1,509 @@
+# 7.1: Creando una Transacción Bitcoin Parcialmente Firmada
+
+> :information_source: **NOTA:** Esta sección se ha agregado recientemente al curso y es un borrador inicial que aún puede estar pendiente de revisión. Lector de advertencias.
```
[the remark this answers is not in the document]
Comment (icculp) [reply es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:3]: This does read better, but there are a bunch of other chapters that use Lector de advertencias that have already been merged. I'll update here though

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 4]
```diff
@@ -0,0 +1,509 @@
...
+
+Inicialmente, los PSBT pueden tener un aspecto similar a los de multifirmas porque tienen un solo bit de funcionalidad superpuesto: la capacidad de firmar conjuntamente una transacción. Sin embargo, fueron creados para un caso de uso totalmente diferente. Los PSBT reconocen la necesidad de que varios programas creen conjuntamente una transacción por varias razones diferentes, y proporcionan un formato regularizado para hacerlo. Son especialmente útiles para casos de uso que involucran carteras de hardware (para lo cual, consulte [§7.3] (https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/blob/master/07_3_Integrating_with_Hardware_Wallets.md )), que están protegidos del acceso total a Internet y tienden a tener un historial de transacciones mínimo.
```
[the remark this answers is not in the document]
Comment (icculp) [reply es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:15]: Second point is clarified after the colon: "the ability to jointly sign a transaction", the "little bit" of functionality they share. Unlikely that functionality could be coded in a single bit 😆

[the remark this answers is not in the document]
Comment (icculp) [reply es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:15]: Fourth bullet, per @javiervargas suggestion I've changed all instances of carteras to billeteras for consistency. But it's still feminine so will correct to protegidas

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 5]
```diff
@@ -1,32 +1,32 @@
...
+Inicialmente, los PSBT pueden tener un aspecto similar a las multifirmas porque tienen un solo bit de funcionalidad superpuesto: la capacidad de firmar conjuntamente una transacción. Sin embargo, fueron creados para un caso de uso totalmente diferente. Los PSBT reconocen la necesidad de que varios programas creen conjuntamente una transacción por varias razones diferentes, y proporcionan un formato regularizado para hacerlo. Son especialmente útiles para casos de uso que involucran billeteras de hardware (para lo cual, consulte [§7.3] (https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/blob/master/07_3_Integrating_with_Hardware_Wallets.md)), que están protegidas del acceso total a Internet y tienden a tener un historial de transacciones mínimo.
```
Comment (csralvall) [inline es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:15]: - Remove space between `[§7.3]` and `(https://github.com/BlockchainCommons/Learning-Bitcoin-from-the-Command-Line/blob/master/07_3_Integrating_with_Hardware_Wallets.md)`. It disables the link rendering.

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 6]
```diff
@@ -500,10 +500,10 @@ Finalmente, si en cambio necesita más control y elige usar `utxoupdatepsbt` (qu
 
 ![](images/psbt-roles-for-cli-3.png)
```
Comment (csralvall) [inline es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:501]: - `` should probably be ``.

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 7]
```diff
@@ -0,0 +1,509 @@
...
+```
+$ psbt_hex=$(bitcoin-cli finalizepsbt $psbt_f | jq -r '.hex')
+$ bitcoin-cli -named sendrawtransaction hexstring=$psbt_hex
+ea73a631b456d2b041ed73bf5767946408c6ff067716929a68ecda2e3e4de6d3
+```
+## Revise el flujo de trabajo
+
+Al crear el software `bitcoin-cli`, lo más probable es que cumpla las cinco funciones principales de los PSBT con `createpsbt`, `walletprocesspsbt` y `finalizepsbt`. Esto es lo que parece ese flujo:
+
+![](images/psbt-roles-for-cli-1.png)
```
Comment (csralvall) [inline es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:493]: - `` should probably be `.`

**Review by csralvall — CHANGES_REQUESTED** [review]
Test if images link changes work as intended.

File: es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md [hunk 8]
```diff
@@ -490,15 +490,15 @@ ea73a631b456d2b041ed73bf5767946408c6ff067716929a68ecda2e3e4de6d3
 
 Al crear el software `bitcoin-cli`, lo más probable es que cumpla las cinco funciones principales de los PSBT con `createpsbt`, `walletprocesspsbt` y `finalizepsbt`. Esto es lo que parece ese flujo:
 
-![](images/psbt-roles-for-cli-1.png)
+![](../images/psbt-roles-for-cli-1.png)
 
 Si elige usar el atajo de `walletcreatefundedpsbt`, así es como se ve en su lugar:
 
 ![](images/psbt-roles-for-cli-2.png)
```
Comment (csralvall) [inline es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md:497]: ``

Comment (icculp) [reply es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md]: sorry I missed that one, great attention to detail!

**Review by csralvall — APPROVED** [review]
@shannona, this is ready to be merged.

**Comment by shannona** [comment]
Thanks.

**Pull request merged**